### PR TITLE
Query And Update Animals Table

### PR DESCRIPTION
**Pull request by peter-akhigbe**
This pull request implements the necessary modifications and queries for the Animals table in accordance with the project requirements. The following changes have been made:

- Added a new column "species" of type string to the Animals table.
- Inserted new animal data with the specified details.
- Implemented transactions for updating the species column, rolling back changes, and verifying the rollback.
- Updated the species column based on specific conditions.
- Performed transactions to delete and rollback records in the Animals table.
- Executed transactions to delete, update, and rollback based on conditions involving animal birth dates and weights.
- Added SQL queries to answer the provided questions related to animal data.

Please review this pull request. Screenshots of the query results and transaction actions have been included in the pull request description for reference.

### CODE REVIEW

**Comment by peter-akhigbe** [comment]
- Before setting the species column to "unspecified".

Screenshot 2023-06-09 at 11 20 23 PM

- After setting the species column to "unspecified".

Screenshot 2023-06-09 at 11 20 28 PM

- ROLLBACK

Screenshot 2023-06-09 at 11 21 03 PM

**Comment by peter-akhigbe** [comment]
Update the animals table by setting the species column to "digimon" for animals with names ending in "mon", and set it to "pokemon" for animals without a species already set.

- Verify changes persist after commit

Screenshot 2023-06-09 at 11 34 10 PM

**Comment by peter-akhigbe** [comment]
Delete all records in the animals table, and then roll back the transaction.

- Verify deletion

Screenshot 2023-06-09 at 11 41 45 PM

- Verify rollback

Screenshot 2023-06-09 at 11 42 04 PM

**Comment by peter-akhigbe** [comment]
- Delete all animals born after Jan 1st, 2022.
- Create a savepoint for the transaction.

Screenshot 2023-06-10 at 12 10 46 AM

- Update all animals' weight to be their weight multiplied by -1.

Screenshot 2023-06-10 at 12 02 36 AM

- Rollback to the savepoint
- Update all animals' weights that are negative to be their weight multiplied by -1.
- Commit transaction
- Verify changes

Screenshot 2023-06-10 at 12 02 57 AM

**Comment by peter-akhigbe** [comment]
# Queries

- How many animals are there?
Screenshot 2023-06-10 at 12 35 30 AM


- How many animals have never tried to escape?
Screenshot 2023-06-10 at 12 37 09 AM


- What is the average weight of animals?
Screenshot 2023-06-10 at 12 37 19 AM


- Who escapes the most, neutered or not neutered animals?
Screenshot 2023-06-10 at 12 37 37 AM


- What is the minimum and maximum weight of each type of animal?
Screenshot 2023-06-10 at 12 38 04 AM


- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
Screenshot 2023-06-10 at 12 38 22 AM

**Review by alexander16108 — APPROVED** [review]
Hi @peter-akhigbe  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



## Highlights
good  pr
proper readme file

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @peter-akhigbe , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

- please fix the broken license link found on your readme file because it is highly unprofessional to have broken links on your codebase

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- please add those screenshots to your PR description, not as comments on the PR to make your PR look neat. 👍🏾 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -256,6 +256,4 @@ I would like to thank Microverse
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (KanzaTahreem) [inline README.md:257]: - [ ] As suggested, please make sure that the LICENSE link is not broken for documenting a professional README. Right now, I am getting the _**File not found**_ error⚠ I can not see a LICENSE file in your repo. Can you please look into this? 😃
  image

**Review by KanzaTahreem — CHANGES_REQUESTED** [review]
# Changes Required 🔁

Hi @peter-akhigbe 🙋‍♀️

Good job so far! I really appreciate the effort you put in!  **_However, there are few things that still need to be fixed. But you are almost there!_** 💪 🏁

so_close

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Great work using correct Gitflow
- ✅ Descriptive PR title & description


## Required Changes 🛑


_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — APPROVED** [review]
Hi @peter-akhigbe 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Professional Readme
✅ Implemented suggested changes
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — COMMENTED** [review]
# Approved :green_circle: 
Hi @peter-akhigbe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Good SQL query design :white_check_mark: 
- [x] Git flow is used correctly :white_check_mark:  
- [x] Pull request has a descriptive title and summary :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**